packages/wrangler/src/__tests__/api-dev.test.ts

```diff
@@ -0,0 +1,17 @@
+import { unstable_dev } from "../api";
+
+jest.unmock("undici");
+
+describe("unstable_dev", () => {
+	it("should return Hello World", async () => {
```

Comment:
This test will fail until we find a solution for https://github.com/cloudflare/wrangler2/issues/1755

Reply:
See https://github.com/cloudflare/wrangler2/pull/1762